docs: add guide for auto-generating commit messages (#67)
Kilo-Org/docs: blog-posts/auto-generate-commit-messages.md

# Why I Let AI Write My Commit Messages
### And Probably You Should Too

> I've just finished implementing a big feature, the code is staged and ready to go, but there I am — mesmerized by the empty commit message field, brain is blank. What do I write? "Fixed stuff"? "Updates"? "asdf"? Do you know that feeling too?

Reviewer: #nit next time _you_  could phrase this with "you" words:

```suggestion
You've just finished implementing a big feature, the code is staged and ready to go, but there you are — mesmerized by the empty commit message field, brain blank. What should you write? "Fixed stuff"? "Updates"? "asdf"? Do you know that feeling?
```
